weather_mv/loader_pipeline/pipeline.py

```diff
@@ -17,15 +17,25 @@
 import datetime
 import json
 import logging
+import tempfile
 import typing as t
 
 import apache_beam as beam
 from apache_beam.io.filesystems import FileSystems
 from apache_beam.options.pipeline_options import PipelineOptions, SetupOptions
+from google.cloud import bigquery, storage
+from google.api_core.exceptions import BadRequest, NotFound
+from urllib.parse import urlparse
 
 from .bq import ToBigQuery
 from .streaming import GroupMessagesByFixedWindows, ParsePaths
 
+CANARY_BUCKET_NAME = 'anthromet_dummy_bucket'
+CANARY_RECORD = {'foo': 'bar'}
+CANARY_RECORD_FILE_NAME = 'dummy_record.json'
+CANARY_OUTPUT_TABLE_SUFFIX = '_anthromet_dummy_table'
+CANARY_TABLE_SCHEMA = [bigquery.SchemaField('name', 'STRING', mode='NULLABLE')]
```

Comment:
Please update the term "dummy" in the strings, too.

Reply:
Done.